terraform/secrets/file.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+env=$1
+secret_name=$2
+file_path=$3
+
+if [ $# -ne 3 ]; then
```

Comment:
I think this check needs to come before referencing `$1`, `$2`, `$3` if they are all required.